bin/validate-declarations.js

```diff
@@ -39,6 +52,10 @@ const VALIDATE_PATH = path.resolve(__dirname, '../scripts/validation/validate.js
 
   let hasFailedTests = false;
 
+  const addValidationTestSuite = (await import('../scripts/declarations/validate/index.mocha.js')).default;
+
+  addValidationTestSuite(program.parse(process.argv).opts());
```

Comment:
`process.argv` is optional here.

```suggestion
  addValidationTestSuite(program.parse().opts());
```